mrc-ide/naomi
# Improve test output

This PR will
* Switch to testthat 3e
* Run tests separately from R CMD check to improve test output in buildkite logs
* Run tests in parallel
